### DISCUSSION TITLE
My Glvis command is killed

### DISCUSSION THREAD

**Discussion by Zhivos** [GLVis/glvis#265] [General]
Hello everyone,

I don't know if someone ever experienced this but my simple command
glvis -np 96 -m mesh -g scalar
gets the error message : "Killed" without any reason of some sort ...

My mesh has approximately 22M nodes

Can somebody help me on that ?

Thanks everyone
Simon

**Comment by tzanio** [comment]
Assuming the files are correct (e.g. you tried the same process at smaller scale and it works), you most likely are running out of memory

**Reply by Zhivos** [reply]
Hello !

Thank you for the very quick answer. I used to open bigger meshes so i'm quite surprised. I'm going to investigate
